### PR TITLE
SAK-42077 :  Can't show a gradebook item when clicking arrows

### PR DESCRIPTION
**Pull request by frasese**

### CODE REVIEW

File: gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js [hunk 1]
```diff
@@ -1255,10 +1255,16 @@ GbGradeTable.modelIndexForStudent = function(studentId) {
 };
 
 
-GbGradeTable.colForAssignment = function(assignmentId) {
-  return GbGradeTable.findIndex(GbGradeTable.instance.view.settings.columns, function(column) {
-           return column._data_ && column._data_.assignmentId === parseInt(assignmentId);
-         });
+GbGradeTable.colForAssignment = function(assignmentId, array) {
+    if(array == undefined){
```
Comment (adrianfish) [inline gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js]: Need a space after if and use ===

Comment (frasese) [reply gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js]: Changes done.
The problem occurs with or without sections. Always when one or more items are hidden at the beginning and some others at the end.

**Comment by frasese** [comment]
@adrianfish would you mind to review the PR again, please.

Thanks

File: gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js [hunk 2]
```diff
@@ -1255,10 +1255,16 @@ GbGradeTable.modelIndexForStudent = function(studentId) {
...
-           return column._data_ && column._data_.assignmentId === parseInt(assignmentId);
-         });
+GbGradeTable.colForAssignment = function(assignmentId, array) {
+    if (array === undefined){
+        return GbGradeTable.findIndex(GbGradeTable.instance.view.settings.columns, function(column) {
+            return column._data_ && column._data_.assignmentId === parseInt(assignmentId);
+        });
+    } else {
+        return GbGradeTable.findIndex(array, function(column) {
+            return column.assignmentId && column.assignmentId === parseInt(assignmentId);
```
Comment (adrianfish) [inline gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js]: I'd add the radix in here, just to be sure, on both parseInts. I know it wasn't there before, but I think it should be there.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description

**Comment by adrianfish** [comment]
Just a small change. Ping me when you've fixed it and I'll approve.

**Comment by frasese** [comment]
Please, use squash + merge

**Pull request merged**